base.html

```diff
@@ -18,15 +18,22 @@
         {{ institution }} {{ session }} Selectives
         </a></h1>
       </td>
+      <!--This message will only display on certain admin pages that pass setup_status.-->
+      {% if setup_status and (setup_status == 'DBUPGRADE') %}
       <td style="color:red">
-        <!--This message will only display on certain admin pages that pass setup_status.-->
-        {% if setup_status and (setup_status == 'FAIL') %}
+        Database upgraded required! Go to Error Check to run upgrade script.
+      </td>
+      {% endif %}
+      {% if setup_status and (setup_status == 'FAIL') %}
+      <td style="color:blue">
```

Comment:
Maybe purple? I want to keep red for only urgent messages. It's possible to use the system without student groups and class groups. If it's always red, people may not notice when there is a vital error.

The problem is not every error is the same importance. Some errors are fatal, like if Dayparts is missing or if the database needs upgrading. But some errors are just good to know, can fix it when we have time.

Maybe we can have FAIL_LEVEL_1 (red: dayparts, students, classes missing), FAIL_LEVEL_2 (purple: student email mismatch), FAIL_LEVEL_3 (blue: student groups missing or mismatch) in addition to OK and UNKNOWN (orange?).

It's beginning to feel like we are programming a nuclear power plant. We could have red flashing for Run Now!